pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>io.burt</groupId>
   <artifactId>jmespath</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>0.2.1-SNAPSHOT</version>
```

Comment:
I think this was a random addition that had to do with me choosing the wrong merge base for the PR. This is no longer part of the PR. Good catch.

You're right that if this gets merged the next release will have to bump the minor release (since we're 0.x.x, it would be a major if we were post 1.0.0). The reason that the version has been bumped at all from 0.2.0 is that that's how Maven's release system works. When you release a version it automatically bumps the minor version of the checked in version number, so that you can't accidentally release SNAPSHOT versions for the version you just released.